content/blog/learn-what-you-can-do-with-hpe-data-services-cloud-console-api-in-just-3-minutes.md

```diff
@@ -0,0 +1,201 @@
+
+![The Data Services Cloud Console API public collection](/img/search-dscc-collection-postman-figure1.png "The Data Services Cloud Console API public collection")
+
+<span style="color:grey; font-family:Arial; font-size:1em">Figure 1: The Data Services Cloud Console API public collection.</span>
+
+You can then fork the collection to make a copy of it in your Postman workspace. You can then work with your own copy of the collection and perform changes without affecting the parent collection. Select the ***more actions*** icon (the ellipsis) next to the collection, then select ***Create a fork***. When you fork the public collection, you can choose to watch the original collection to be notified about changes made to the parent collection. This allows you to pull updates from the parent collection into your forked copy, should the parent collection be updated.
+
+Alternatively, you can export the public collection locally to your local storage as JSON file and import it as a new collection in your Postman workspace. 
+
+> **Note:** You must sign in to your Postman account to create a fork or export the public collection. To fork a collection within a public workspace, you must also enable your public profile in your Postman profile settings.  
+
+### Step 3 – Set your Data Services Cloud Console collection variables
+
+The Data Services Cloud Console API collection built by Mark makes use of collection variables that are available throughout the REST API requests in the collection. Select the collection in your workspace and then select the ***Variables*** tab as shown below.
+
+![HPE Data Services Cloud Console collection variables](/img/variables-dscc-collection-postman-figure2.png "HPE Data Services Cloud Console collection variables")
+
+<span style="color:grey; font-family:Arial; font-size:1em">Figure 2: HPE Data Services Cloud Console collection variables.</span>
+
+Define the Current Value of the collection variables to match your Data Services Cloud Console context:    
+
+* **baseUrl**: This variable defines the base URL of the REST API requests. It should match the regional endpoint of your Data Services Cloud Console application instance where your storage devices are registered.    
+
+* **ClientId** and **ClientSecret**: they should be set with the value of your Client Application API credentials you previously created using the HPE GreenLake Cloud Platform GUI. These variables are used to request an OAuth access token by authenticating with the authorization server referenced in the **sso_URI** variable.    
+
+* **sso_URI**: This variable is the URI of the OAuth authorization server. If your organization has set up their own HPE GreenLake SAML Single Sign-On (SSO) authorization server to create access token, replace the current default value with your SSO URI. Otherwise keep the value for this variable as currently set to *sso.common.cloud.hpe.com/as/token.oauth2*.    
```

Comment:
The word "an" is missing between "...server to create" and "access token, replace..."